internal/capabilities/capabilities_test.go

```diff
@@ -0,0 +1,110 @@
+package capabilities
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+)
+
+func TestLookupFromFile(t *testing.T) {
+	t.Parallel()
+
+	// Test that we are able to load a capabilities file using a file://
+	// URL.
+
+	path, err := filepath.Abs("./testdata/capabilities.json")
+	if err != nil {
+		t.Fatalf("could not determine absolute path to test capabilities file: %v", err)
+	}
+
+	urlForPath := "file://" + path
+
+	caps, err := Lookup(context.Background(), urlForPath)
+	if err != nil {
+		t.Errorf("unexpected error from Lookup: %v", err)
+	}
+
+	if len(caps.Builtins) != 1 {
+		t.Errorf("expected capabilities to have exactly 1 builtin")
+	}
+
+	if caps.Builtins[0].Name != "unittest123" {
+		t.Errorf("builtin name is incorrect, expected 'unittest123' but got '%s'", caps.Builtins[0].Name)
+	}
+}
+
+func TestLookupFromURL(t *testing.T) {
+	t.Parallel()
+
+	// Test that we can load a one of the existing OPA capabilities files
+	// via GitHub.
+
+	caps, err := Lookup(
+		context.Background(),
+		"https://raw.githubusercontent.com/open-policy-agent/opa/main/capabilities/v0.55.0.json",
```

Comment:
This test should probably go in a separate file with a tag, like "integration", as I don't think unit tests should require an internet connection.

Reply:
b5cb5c5